client/client.go

```diff
@@ -567,14 +567,23 @@ func DsLookup(query dsQuery) (string, string, error) {
 // Dial implements ssh.Dial for cpu.
 // Additionaly, if Cmd.Root is not "", it
 // starts up a server for 9p requests.
+// Note that any bind parsing is deferred until this point,
```

Comment:
i guess Dial() is the first thing called after the cmd args parsing?  wonder if there's a more explicit "finished with args" function we can call, in case later on we do something between the arg parsing and Dial.

Reply:
I figure Dial is safe, it basically means "time to talk to cpud" and that's a good time to assemble the environment variables.